### PR TITLE
Update requirements

### PR DESCRIPTION
**Pull request by DonRichards**
Added a link to a ticket that includes more information on getting composer to work correctly.
Removed drush because these requirements are for the local machine and not for what's in the container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,13 +37,12 @@ as database import/export and reindexing.
 
 ## Requirements
 
-- Composer 1.10+
+- Composer 1.10+ (see [issues/184](https://github.com/Islandora-Devops/isle-dc/issues/184) for help). 2.x currently does not work.
```
Comment (nigelgbanks) [inline README.md]: The comment here is about `docker-compose` not PHP's `composer` tool.

File: README.md [hunk 2]
```diff
@@ -37,13 +37,12 @@ as database import/export and reindexing.
...
 ## Requirements
 
-- Composer 1.10+
+- Composer 1.10+ (see [issues/184](https://github.com/Islandora-Devops/isle-dc/issues/184) for help). 2.x currently does not work. 
 - Desktop / laptop / VM (*Docker must have sufficient resources to run GNU Make*)
 - Docker-CE 19.x+ (*If using Docker Desktop for Windows, any stable release
   *after* 2.2.0.4, or use a 2.2.0.4 with a [patch][Docker for Windows Patch] due
   to a [bug][Docker for Windows Bug]*)
 - Docker-compose version 1.25.x+
-- Drush 9.0+
```
Comment (nigelgbanks) [inline README.md:46]: Good catch.

**Review by nigelgbanks — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DonRichards) [reply README.md]: :facepalm: Noted, I'll make that adjustment.

Comment (DonRichards) [reply README.md]: @nigelgbanks OK, got it.

**Comment by DonRichards** [comment]
@nigelgbanks Good to merge?

**Comment by jordandukart** [comment]
Just an FYI @DonRichards this is merge conflicting.

**Comment by nigelgbanks** [comment]
Good stuff

**Pull request merged**